webpack.config.ts uses double quotes instead of single and fails lint

I had fresh clone of the repo and was attempting to add new feature and it push it and it failed due to error in webpack.config.ts

```
E:\Repos\ts-test-package [master ↑2 +0 ~2 -0 !]> npm run lint

> @mattmazzola/ts-test-package@0.0.0 lint E:\Repos\ts-test-package
> tslint -e **/node_modules/** -e **/dist/** **/*.ts


webpack.config.ts[3, 25]: " should be '

npm ERR! Windows_NT 10.0.14393
```

--- a/webpack.config.ts
+++ b/webpack.config.ts
@@ -1,6 +1,6 @@
 import { join, resolve } from 'path'
 const { camelCase } = require('lodash')
-const webpack = require("webpack")
+const webpack = require('webpack')
 const { TsConfigPathsPlugin, CheckerPlugin } = require('awesome-typescript-loader')
 const TypedocWebpackPlugin = require('typedoc-webpack-plugin')
 const HtmlWebpackPlugin = require('html-webpack-plugin')

